helm-charts/helm3/strimzi-kafka-operator/values.yaml

```diff
@@ -167,4 +167,4 @@ labelsExclusionPattern: ""
 # Controls whether Strimzi generates network policy resources (By default true)
 generateNetworkPolicy: true
 # Override the value for Connect build timeout
-connectBuildTimeoutMs: 300000
+connectBuildTimeoutMs: 300000
```

Comment:
The changes look better now. But you would need to revert this new-line change to get the build to pass.